2021-05-01 18:12:21,696 : INFO : Start of Prediction
2021-05-01 18:12:21,696 : INFO : run_id:2021-05-01_181221_734616497
2021-05-01 18:12:21,696 : INFO : Start of Data Load, validation and transformation
2021-05-01 18:12:21,696 : INFO : Start of Archiving Old Rejected Files...
2021-05-01 18:12:21,696 : INFO : End of Archiving Old Rejected Files...
2021-05-01 18:12:21,697 : INFO : Start of Archiving Old Validated Files...
2021-05-01 18:12:21,748 : INFO : End of Archiving Old Validated Files...
2021-05-01 18:12:21,748 : INFO : Start of Archiving Old Processed Files...
2021-05-01 18:12:21,749 : INFO : End of Archiving Old Processed Files...
2021-05-01 18:12:21,749 : INFO : Start of Archiving Old Result Files...
2021-05-01 18:12:21,751 : INFO : End of Archiving Old Result Files...
2021-05-01 18:12:21,751 : INFO : Start of Reading values From Schema...
2021-05-01 18:12:21,751 : INFO : End of Reading values From Schema...
2021-05-01 18:12:21,752 : INFO : Start of Validating Column Length...
2021-05-01 18:12:21,752 : INFO : End of Validating Column Length...
2021-05-01 18:12:21,752 : INFO : Start of Validating Missing Values...
2021-05-01 18:12:21,752 : INFO : End of Validating Missing Values...
2021-05-01 18:12:21,752 : INFO : Start of Replacing Missing Values with NULL...
2021-05-01 18:12:21,753 : INFO : End of Replacing Missing Values with NULL...
2021-05-01 18:12:21,753 : INFO : Start of Creating Table...
2021-05-01 18:12:21,753 : INFO : Opened prediction database successfully
2021-05-01 18:12:21,951 : INFO : CREATE TABLE prediction_raw_data_t column_name
2021-05-01 18:12:22,017 : INFO : ALTER TABLE prediction_raw_data_t ADD COLUMN
2021-05-01 18:12:22,139 : INFO : ALTER TABLE prediction_raw_data_t ADD COLUMN
2021-05-01 18:12:22,238 : INFO : ALTER TABLE prediction_raw_data_t ADD COLUMN
2021-05-01 18:12:22,339 : INFO : ALTER TABLE prediction_raw_data_t ADD COLUMN
2021-05-01 18:12:22,402 : INFO : ALTER TABLE prediction_raw_data_t ADD COLUMN
2021-05-01 18:12:22,471 : INFO : ALTER TABLE prediction_raw_data_t ADD COLUMN
2021-05-01 18:12:22,537 : INFO : ALTER TABLE prediction_raw_data_t ADD COLUMN
2021-05-01 18:12:22,604 : INFO : ALTER TABLE prediction_raw_data_t ADD COLUMN
2021-05-01 18:12:22,604 : INFO : End of Creating Table...
2021-05-01 18:12:22,619 : INFO : Opened prediction database successfully
2021-05-01 18:12:22,620 : INFO : Start of Inserting Data into Table...
2021-05-01 18:12:22,620 : INFO : End of Inserting Data into Table...
2021-05-01 18:12:22,620 : INFO : Start of Exporting Data into CSV...
2021-05-01 18:12:22,620 : INFO : Opened prediction database successfully
2021-05-01 18:12:22,622 : INFO : End of Exporting Data into CSV...
2021-05-01 18:12:22,622 : INFO : Start of Moving Processed Files...
2021-05-01 18:12:22,623 : INFO : End of Moving Processed Files...
2021-05-01 18:12:22,623 : INFO : End of Data Load, validation and transformation
2021-05-01 18:12:22,623 : INFO : Start of Preprocessing...
2021-05-01 18:12:22,623 : INFO : Start of reading dataset...
2021-05-01 18:12:22,636 : INFO : End of reading dataset...
2021-05-01 18:12:22,636 : INFO : Start of feature encoding...
2021-05-01 18:12:22,685 : INFO : End of feature encoding...
2021-05-01 18:12:22,686 : INFO : Start of Droping Columns...
2021-05-01 18:12:22,688 : INFO : End of Droping Columns...
2021-05-01 18:12:22,688 : INFO : Start of finding missing values...
2021-05-01 18:12:22,690 : INFO : End of finding missing values...
2021-05-01 18:12:22,690 : INFO : Start of building final predictset...
2021-05-01 18:12:22,706 : INFO : End of building final predictset...
2021-05-01 18:12:22,706 : INFO : End of Preprocessing...
2021-05-01 18:12:22,706 : INFO : Start of Load Model
2021-05-01 18:12:22,706 : INFO : Model File KMeans loaded
2021-05-01 18:12:22,706 : INFO : End of Load Model
2021-05-01 18:12:22,712 : ERROR : Unsuccessful End of Prediction
Traceback (most recent call last):
  File "D:\GitHub\Employee-Retention-Prediction\App\apps\prediction\predict_model.py", line 31, in batch_predict_from_model
    clusters = kmeans.predict(self.X)
  File "D:\Programming\Python\Anaconda\lib\site-packages\sklearn\cluster\_kmeans.py", line 1201, in predict
    X = self._check_test_data(X)
  File "D:\Programming\Python\Anaconda\lib\site-packages\sklearn\cluster\_kmeans.py", line 993, in _check_test_data
    X = check_array(X, accept_sparse='csr', dtype=[np.float64, np.float32],
  File "D:\Programming\Python\Anaconda\lib\site-packages\sklearn\utils\validation.py", line 72, in inner_f
    return f(**kwargs)
  File "D:\Programming\Python\Anaconda\lib\site-packages\sklearn\utils\validation.py", line 650, in check_array
    raise ValueError("Found array with %d sample(s) (shape=%s) while a"
ValueError: Found array with 0 sample(s) (shape=(0, 11)) while a minimum of 1 is required.
